src/utils/pgwire/src/pg_protocol.rs

```diff
@@ -114,16 +114,21 @@ pub struct TlsConfig {
 
 impl TlsConfig {
     pub fn new_default() -> Self {
-        let cert = PathBuf::new().join("tests/ssl/demo.crt");
-        let key = PathBuf::new().join("tests/ssl/demo.key");
-        let path_to_cur_proj = PathBuf::new().join("src/utils/pgwire");
+        let cert = if let Ok(cert) = std::env::var("RW_SSL_CERT") {
+            PathBuf::from(cert)
+        } else {
+            PathBuf::new().join("src/utils/pgwire").join("tests/ssl/demo.crt")
```

Comment:
Should this be test-only? If so, should we instead pass `None` here if they're unset?

https://github.com/risingwavelabs/risingwave/blob/6286ddd2a4447bc0ab8608c81ad9480b2e115752/src/frontend/src/lib.rs#L161

Reply:
According to #5775  @wangrunji0408 do you know whether we still use SSL in deterministic tests?

Reply:
If it's believe okay to use "demo" cert in production, then we may consider embedding the contents with `include_bytes` and create a temp file at runtime.

Reply:
If we don't use it anymore, I will remove those test purpose codes.

Reply:
> According to #5775 @wangrunji0408 do you know whether we still use SSL in deterministic tests?

I remember SSL is always disabled in deterministic tests.